### PR TITLE
Push Notifications packaged sample

### PR DESCRIPTION
**Pull request by loneursid**
Packaged version of the Push Notification sample.

Created as a delta of the unpackaged sample as both specializations of the same sample base.

### CODE REVIEW

File: Samples/PushNotifications/cpp-console/cpp-console.cpp [hunk 1]
```diff
@@ -0,0 +1,199 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include <windows.h>
+#include <iostream>
+
+//#include <unknwn.h>
```
Comment (danielayala94) [inline Samples/PushNotifications/cpp-console/cpp-console.cpp:7]: Can we just remove the headers we don't need?

File: Samples/PushNotifications/cpp-console/cpp-console.cpp [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+using namespace winrt::Windows::ApplicationModel::Activation;
+using namespace winrt::Windows::ApplicationModel::Background; // BackgroundTask APIs
+using namespace winrt::Windows::Foundation;
+//using namespace winrt::Windows::Storage;
+//using namespace winrt::Windows::Storage::Streams;
+using namespace winrt::Windows::Globalization::DateTimeFormatting;
+
+// To obtain an AAD RemoteIdentifier for your app,
+// follow the instructions on https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app
+winrt::guid remoteId{ "00000000-0000-0000-0000-000000000000"}; // Replace this with own remoteId
```
Comment (danielayala94) [inline Samples/PushNotifications/cpp-console/cpp-console.cpp:41]: Please address the comments from the unpkg sample PR applicable to this PR.

File: Samples/PushNotifications/cpp-console/cpp-console.cpp [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+            std::cout << "Push notification content received from FOREGROUND: " << payloadString << std::endl;
+
+            // Set handled to true to prevent background activation
+            args.Handled(true);
+        });
+}
+
+int main()
+{
+    if (PushNotificationManager::IsActivatorSupported(PushNotificationRegistrationActivators::ComActivator))
```
Comment (danielayala94) [inline Samples/PushNotifications/cpp-console/cpp-console.cpp:123]: Should this be?

```suggestion
    if (PushNotificationManager::IsActivatorSupported(PushNotificationRegistrationActivators::PushTrigger | PushNotificationRegistrationActivators::ComActivator))
```

File: Samples/PushNotifications/cpp-console/cpp-console.cpp (hunk 2 above)
Comment (loneursid) [reply Samples/PushNotifications/cpp-console/cpp-console.cpp]: All feedback made into the unpkg sample PR up to now, have equaly been addressed in this PR.

File: Samples/PushNotifications/cpp-console/cpp-console.cpp [hunk 4]
```diff
@@ -133,63 +118,62 @@ int main()
...
+            auto deferral = pushArgs.GetDeferral();
 
-        // Call GetDeferral to ensure that code runs in low power
-        auto deferral = pushArgs.GetDeferral();
+            auto payload = pushArgs.Payload();
 
-        auto payload = pushArgs.Payload();
+            // Do stuff to process the raw notification payload
+            std::string payloadString(payload.begin(), payload.end());
+            std::cout << std::endl << "Push notification content received from BACKGROUND: " << payloadString.c_str() << std::endl;
```
Comment (danielayala94) [inline Samples/PushNotifications/cpp-console/cpp-console.cpp:160]: Similar as in the unpkg sample PR. Any reason for `std::cout << std::endl << "Text"` ?

Comment (loneursid) [reply Samples/PushNotifications/cpp-console/cpp-console.cpp]: Similar answer. I've replaced them with \n.

**Comment by loneursid** [comment]
Closing, there's a new PR for this sample

**Pull request closed without merging**